pages/contact.tsx

```diff
@@ -0,0 +1,6 @@
+import { NextPage } from "next";
+import ContactForm from "components/ContactForm";
+const contact: NextPage = () => {
+  return <ContactForm></ContactForm>;
```

Comment:
The Contact Form component needs to be placed within the layout component here.

Reply:
I've added the layout component, but need to fix some styling before committing this PR. The header/footer cuts off the contact form. Will commit a fix soon!